api/localDevAuth.ts

```diff
@@ -6,6 +6,8 @@ import {
   ScopeData,
   AccessTokenResponse,
   EnabledFeaturesResponse,
+  ScopeGroupAuthorization,
```

Comment:
nit: looks like this import isn't being used

Reply:
ty! good catch